scripts/quests/otherAreas/For_the_Birds.lua

```diff
@@ -0,0 +1,211 @@
+-----------------------------------
+-- For the Birds
+-----------------------------------
+-- Log ID: 4, Quest ID: 104
+-- Koblakiq          !pos -64 21 -117 11
+-- Daa Bola the Seer !pos -157 -17 193 151
+-- GeFhu Yagudoeye   !pos -91 -3 -127 147
+-----------------------------------
+local oztrojaID  = zones[xi.zone.CASTLE_OZTROJA]
+local beadeauxID = zones[xi.zone.BEADEAUX]
+-----------------------------------
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.FOR_THE_BIRDS)
+
+quest.reward =
+{
+    item = xi.item.JAGUAR_MANTLE,
+}
+
+local mobListTable =
```

Comment:
You have the offset, and know you're going to go from 0 to 3 every time, no need to explicitly define the table.  Instead, it would be cleaner for the "AllDead" function to iterate `for mobId = beadeauxID.mob.FOR_THE_BIRDS_MOB_OFFSET, beadeauxID.mob.FOR_THE_BIRDS_MOB_OFFSET + 3`

Reply:
Done ✅

Changed the for loop, left the table for the popFromQM call